docs/controls/access_control/ac-05.md

```diff
@@ -0,0 +1,52 @@
+
+c. Defines information system access authorizations to support separation of duties.
+
+### TDP Implementation
+
+**Part a.**  
+Software development and maintenance is separated through the following:
+* Developers use GitHub for project planning as well as the hosting site for all code repositories.  Additionally, Circle CI is used for Continuous Integration / Continuous Delivery - it automates checks for making sure all automated testing, linting and vulnerability scanning checks pass before committing code to the repository or deploying to any environment.  Finally, Docker is used to set up a code based container environment to ensure the same environment is used during development as it is when deployed to the cloud.  Docker also automates the deploys themselves.  Currently, only a dev environment is in place. A staging environment (which could be called a test environment) and a production environment will be set up in future releases.
+* System owner manages the account access. Additionally, System owner provisions a new user account on request.
+
+Data creation, control, and security functions are completed through the TDP application by a user who has been authenticated and is active in the system. These roles include OFA admin and STT users. 
+* STT/Data Preppers collect TANF data and transmit final data to OFA through the TANF application.
+* OFA admins have the ability to upload data on behalf of STTs and upload data files locally into the web application. 
+* These users cannot provision new accounts and do not have access to the account management functionality.
+
+**Part b.**  
+
+**Developers**  
+  - Use GitHub for project planning as well as the hosting site for all code repositories.  
+  - Ensures all testing, linting and vulnerability scan checks are passed before committing code to the repository or deploy to any environment.  
+  - Ensures a consistent development environment by using Docker to set up a code based container 
+  - Develops code in Dev environment.  
+  - Currently, only a dev environment is in place. A staging environment (which could be called a test environment) and a production environment will be set up in future releases.  The process of dev, stage (testing) and release to production is planned for future releases.  However, all code and documentation committed to HHS has to be approved by the Product Manager and the HHS Tech Lead.  
+
+**Users (OFA Admin and STTs)**
+  - approves new users, updates profile information, inactivates, and reactivates users
+  - reads data over assigned region
+  - able to upload and submit new TANF reports, replace and resubmit TANF reports  
+  - access (to be defined) allows the development of reports as required
+
+**Part c.**   
+Granting access to the TDRS system is performed by the OFA Admin through a request in the application.  Users request access to be able to create a profile and submit/view report in the application.
```

Comment:
1. TDRS should be TDP
2. Users create a profile as part of the process to request access. They don't need access to create a profile.

Reply:
Updated to reflect the comment/feedback.